### ISSUE TITLE
Bind the LEFT or RIGHT side of the SUPER key

### ISSUE THREAD

**Issue by CXCubeHD** [hyprwm/Hyprland#8378]
### Description

From what I know there is currently no way to only bind the left or right side of the `SUPER` key. 

Pls correct me if I am wrong >_<

**Comment by vaxerski** [comment]
as a key or mod?

**Comment by CXCubeHD** [comment]
> as a key or mod?

mod

**Comment by vaxerski** [comment]
yeah you cant, might be possible with combos tho: https://wiki.hyprland.org/Configuring/Binds/#keysym-combos

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
